frontend/src/views/WheresMyGene/components/Main/index.tsx

```diff
@@ -284,7 +280,7 @@ export default function WheresMyGene(): JSX.Element {
       </Head>
 
       <SideBar
-        label={<SideBarLabel>Filters</SideBarLabel>}
```

Comment:
Because the sidebar toggle button's font-weight is a standardised `600` we no longer need the wrapper `SideBarLabel`.